robot-db/src/main/java/dao/CategoryDAO.java

```diff
@@ -0,0 +1,14 @@
+package dao;
+
+import domain.Category;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Component;
+
+/**
+  * This class is responsible for managing Categorise in database
+ */
+
+@Component
+public interface CategoryDAO extends CrudRepository<Category, Long> {
+
```

Comment:
No extra lines between Javadocs and class it details please. Also, don't write Javadocs for X starting with "This X is responsible". Javadocs are meant for other devs. They know if it's a class, a method, a package or a field. Inform them of important matters, not of all matters.